src/webaudio.js

```diff
@@ -257,6 +257,7 @@ WaveSurfer.WebAudio = {
         this.gainNode.disconnect();
         this.scriptNode.disconnect();
         this.analyser.disconnect();
+        this.ac.close();
```

Comment:
Something like this: https://github.com/katspaugh/wavesurfer.js/pull/862/commits/3c9a2bab729c75488814039a99084c1815bc9029